graspologic/utils/utils.py

```diff
@@ -701,3 +705,51 @@ def plug_in_variance_estimator(x):
         return covariances
 
     return plug_in_variance_estimator
+
+
+def remap_labels(y_true, y_pred, return_map=False):
```

Comment:
Add type hints. This should be pretty simple for this function.

See [this](https://mypy.readthedocs.io/en/stable/cheat_sheet_py3.html) for a quick guide.

Reply:
done. though I need to check mypy output. do you know how to do typing for variable output possibilities? I looked around and couldn't find anything. not sure if it's important, I think the main use case is just getting the array back in this PR. 

also, this code should technically work on anything sklearn considers "array-like" which I assume is a massive list of stuff. For now I just put list, array, series since again, I think this is 99.9% of use cases

Reply:
Union[type1, type2, type3]

Reply:
https://docs.python.org/3/library/typing.html#typing.Union

Reply:
@dwaynepryce i was just talking about what stuff goes in the Union[]

Reply:
since "array-like" is vague and I'm sure a lot of stuff would actually work